Pidaq/Examples/senderInfo.json

```diff
@@ -0,0 +1,28 @@
+{
+  "canInfo": {
+    "serialPort": "COM5",
+    "busType": "slcan",
+    "bitrate": 500000,
+    "busFilters": [],
+    "messages": [
```

Comment:
See my comment on receiverInfo.json for how python-can interprets the integer values